diff --git a/node_modules/expo-constants/android/build.gradle b/node_modules/expo-constants/android/build.gradle
index fcc166a..179c706 100644
--- a/node_modules/expo-constants/android/build.gradle
+++ b/node_modules/expo-constants/android/build.gradle
@@ -1,6 +1,6 @@
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
-apply plugin: 'maven'
+apply plugin: 'maven-publish'
 
 group = 'host.exp.exponent'
 version = '11.0.2'
@@ -27,7 +27,7 @@ configurations {
 
 //Creating sources with comments
 task androidSourcesJar(type: Jar) {
-  classifier = 'sources'
+  archiveClassifier = 'sources'
   from android.sourceSets.main.java.srcDirs
 }
 
@@ -36,11 +36,10 @@ artifacts {
   archives androidSourcesJar
 }
 
-uploadArchives {
-  repositories {
-    mavenDeployer {
-      configuration = configurations.deployerJars
-      repository(url: mavenLocal().url)
+publishing {
+  publications {
+    maven(MavenPublication) {
+      artifact androidSourcesJar
     }
   }
 }
