diff --git a/node_modules/expo-image-loader/android/build.gradle b/node_modules/expo-image-loader/android/build.gradle
index 7190840..d2982db 100644
--- a/node_modules/expo-image-loader/android/build.gradle
+++ b/node_modules/expo-image-loader/android/build.gradle
@@ -1,6 +1,6 @@
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
-apply plugin: 'maven'
+apply plugin: 'maven-publish'
 
 buildscript {
   // Simple helper that allows the root project to override versions declared by this library.
@@ -36,15 +36,13 @@ artifacts {
   archives androidSourcesJar
 }
 
-uploadArchives {
-  repositories {
-    mavenDeployer {
-      configuration = configurations.deployerJars
-      repository(url: mavenLocal().url)
+publishing {
+  publications {
+    maven(MavenPublication) {
+      artifact androidSourcesJar
     }
   }
 }
-
 android {
   compileSdkVersion safeExtGet("compileSdkVersion", 30)
 
